docs/embrace-api/supported_metrics_and_queries.md

```diff
@@ -44,76 +45,62 @@ However, we strongly recommend transitioning to the new metrics to ensure a cons
+## Sample PromQL queries
 
+* Sessions grouped by devices for a given app version
 ```promql
-sum(daily_sessions_total{app_id="<app ID>"}) by (app_version)
+sum(daily_sessions_total{app_id="<app ID>", app_version="1.2.3"}) by (device_model)
 ```
 
-#### Sessions Grouped by Devices
-
+* Percentage of crash free sessions by devices.
 ```promql
-sum(daily_sessions_total{app_id="<app ID>"}) by (device_model)
+1 - sum(hourly_crashes_total{app_id="$app_id"}) by (device_model) / sum(hourly_sessions_total{app_id="$app_id"}) by (device_model) * 100
 ```
 
-#### Sessions Grouped by Devices for a Given App Version
-
+* Percentage of crash sessions by devices.
 ```promql
-sum(daily_sessions_total{app_id="<app ID>", app_version="1.2.3"}) by (device_model)
+sum(hourly_crashes_total{app_id="$app_id"}) by (device_model) / sum(hourly_sessions_total{app_id="$app_id"}) by (device_model) * 100
 ```
 
-#### Percentage of crash free sessions
+Also, you can pull data for one, multiple, or all of your organization's apps in a single query.
 
+* To pull for a single app, include the `app_id` in the PromQL filter,
 ```promql
-(1 - (sum(hourly_crashes_total{app_id="$app_id"}) / sum(hourly_sessions_total{app_id="$app_id"}) )) * 100
+sum(hourly_custom_metric_sessions_total{app_id="a1b2C3"})
 ```
 
-#### Percentage of crash free sessions by Devices
-
+* To pull for multiple apps, include a pipe-delimited array in the filter,
 ```promql
-1 - sum(hourly_crashes_total{app_id="$app_id"}) by (device_model) / sum(hourly_sessions_total{app_id="$app_id"}) by (device_model) * 100
+sum(hourly_custom_metric_sessions_total{app_id=~"a1b2C3|Z9Y8x7"}) 
 ```
 
-#### Percentage of crash sessions by Devices
-
+* To pull for all apps, do not include any app ID in the filter,
 ```promql
-sum(hourly_crashes_total{app_id="$app_id"}) by (device_model) / sum(hourly_sessions_total{app_id="$app_id"}) by (device_model) * 100
+sum(hourly_custom_metric_sessions_total{})
 ```
-## Custom Metrics
 
-Users may request Custom Metrics via their Customer Success Manager. These will be available via the Metrics API and can
-also be forwarded to your organization's [observability platform of choice](/data-destinations).
+## Dimension reduction - "Other"
 
-### Sample Queries
+To reduce storage costs with various observability platforms (eg Datadog), Embrace Metrics examine high cardinality dimensions for consolidation.
 
-You can pull data for one, multiple, or all of your organization's apps in a single query.
+### Device Models
+There are over 40,000 unique device models on the Android operating system.  The bottom 39,000 models account for ~30% of data typically.  Aside from being expensive to store this many unique values, it is also unwieldy to visualize or review!
+
+<img src={require('@site/static/images/embrace-api/device_other.png').default} alt="Chart showing data by device ranking" />
+
+Currently, we roll together these long-tail device models into an "other" value.
+
+## Metrics Availability
```

Comment:
@scottieb3 I added this graph to explain when the metrics will be available. Could you read it and check if it makes sense and it is clear enough?

Reply:
Thank you